tutorials/speaker_tasks/Speaker_Diarization_Inference.ipynb

```diff
@@ -386,7 +386,9 @@
             "source": [
                 "Note in this tutorial, we use the VAD model MarbleNet-3x2 introduced and published in [ICASSP MarbleNet](https://arxiv.org/pdf/2010.13886.pdf). You might need to tune on dev set similar to your dataset if you would like to improve the performance.\n",
                 "\n",
-                "And the speakerNet-M-Diarization model achieves 7.3% confusion error rate on CH109 set with oracle vad. This model is trained on voxceleb1, voxceleb2, Fisher, SwitchBoard datasets. So for more improved performance specific to your dataset, finetune speaker verification model with a devset similar to your test set."
+                "And the speakerNet-M-Diarization model achieves 7.3% confusion error rate on CH109 set with oracle vad. This model is trained on voxceleb1, voxceleb2, Fisher, SwitchBoard datasets. So for more improved performance specific to your dataset, finetune speaker verification model with a devset similar to your test set.\n",
+		"\n",
+		"Python mulitprocessing with jupyter notebook would hang for a couple cases. In this tutorial, we set num_workers=1, please use script in `NeMo/examples/speaker_tasks/` with num_workers larger than 1 for faster process."
```

Comment:
Could you change this a bit as follows?: 
It is recommended to set `num_workers=1` since using `mulitprocessing` package in Jupyter Notebook environment might cause freezing issues. For sizable data, run speaker diarization using the scripts in NeMo/examples/speaker_tasks/ setting `num_workers` larger than 1 in the configurations.